index.html

```diff
@@ -0,0 +1,186 @@
+                  learn about the latest trends in the world.
+                </p>
+              </div>
+            </div>
+            <div class="program-card">
+              <div class="program-icon">
+                <img
+                  src="./images/program/program_icon_02.png"
+                  alt="CC Exhibition"
+                />
+              </div>
+              <div class="program-sub-title">CC Exhibition</div>
+              <div class="program-sub-description">
+                <p>
+                  Let's meet the creations of artists from various fields who
+                  share the open spirit of CC.
+                </p>
+              </div>
+            </div>
+            <div class="program-card">
+              <div class="program-icon">
+                <img src="./images/program/program_icon_03.png" alt="Forum" />
+              </div>
+              <div class="program-sub-title">Forum</div>
+              <div class="program-sub-description">
+                <p>
+                  We have time to share our thoughts and opinions with experts
+                  for each topic.
+                </p>
+              </div>
+            </div>
+            <div class="program-card">
+              <div class="program-icon">
+                <img
+                  src="./images/program/program_icon_04.png"
+                  alt="Workshop"
+                />
+              </div>
+              <div class="program-sub-title">Workshop</div>
+              <div class="program-sub-description">
+                <p>
+                  You can create your own creations using open source instead of
+                  just looking at them.
+                </p>
+              </div>
+            </div>
+            <div class="program-card">
+              <div class="program-icon">
+                <img
+                  src="./images/program/program_icon_05.png"
+                  alt="CC Ignite"
+                />
+              </div>
+              <div class="program-sub-title">CC Ignite</div>
+              <div class="program-sub-description">
+                <p>
+                  Create opportunities to freely network with CC personnel from
+                  around the world to ignite your ideas
+                </p>
+              </div>
+            </div>
+          </div>
+          <button class="join-button" type="button">
+            Join CC Global Summit 2015
+          </button>
+        </section>
+      </div>
+    </section>
+
+    <section class="wrap speakers-wrap b-bottom clearfix">
+      <div class="content">
+        <h3>Featured Speakers</h3>
+        <div class="guide-bar"></div>
+        <div class="speakers-list" id="speakers-list"></div>
+        <button class="more-btn" type="button" id="more-btn">
+          MORE <i class="fa-solid fa-arrow-right"></i>
+        </button>
+      </div>
+    </section>
+    <script src="./js/index.js"></script>
+  </body>
```

Comment:
- [ ]  Please make sure that the mobile version looks exactly like Behance design make sure your website is responsive 

| Your design | Behance |
| ------------- | --------- |
| ![image](https://user-images.githubusercontent.com/74744787/163299227-0d8842a7-390c-4118-9ee1-3c091f986aba.png)
![image](https://user-images.githubusercontent.com/74744787/163299301-eedc3cd9-347c-450f-858e-5fbd2a3b6be7.png) 
![image](https://user-images.githubusercontent.com/74744787/163299345-a4704c58-58ec-491b-adfc-7659c65a679e.png) | ![image](https://user-images.githubusercontent.com/74744787/163299387-85662614-aec3-4e07-8800-98b432eb41d1.png) ![image](https://user-images.githubusercontent.com/74744787/163299426-93765af7-cbea-47cf-b91d-c00dd9bd1532.png) |

- [ ] Please add the hovering to your button here